### ISSUE TITLE
How can I disable "Start Stop Timer" while recording?

### ISSUE THREAD

**Issue by jimmycodechef** [ShareX/ShareX#3178]
I am unable to find the way to disable "Start Stop Timer" while recording gif or video. Due to this issue, I fail to see options inside taskbar when I am recording full page screencast.

<a href="http://tinypic.com?ref=29m97a9" target="_blank"><img src="http://i66.tinypic.com/29m97a9.jpg" border="0" alt="Image and video hosting by TinyPic"></a>

**Comment by anhiliate** [comment]
+1

**Comment by dragus0** [comment]
+1

**Comment by dragus0** [comment]
or at least position it on the right side

**Comment by ppau2004** [comment]
Agree. I am not able to see or click on anything behind this timer bar.
This bar show be made optional and by default hidden. No one really wants to see it all the time during the recording.

![image](https://user-images.githubusercontent.com/32998047/153710136-3391dcd9-6828-47bc-80e5-95a1b6b850e6.png)
